components/maps/README.md

```diff
@@ -16,58 +18,67 @@ npm install @syncfusion/ej2-maps
 
 ## Resources
 
-* [Getting Started](https://ej2.syncfusion.com/angular/documentation/maps/getting-started.html)
-* [View Online Demos](https://ej2.syncfusion.com/angular/demos/#/material/maps/default)
-* [Product Page](https://www.syncfusion.com/products/angular/maps)
+* [Getting Started](https://ej2.syncfusion.com/angular/documentation/maps/getting-started.html?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm)
+* [View Online Demos](https://ej2.syncfusion.com/angular/demos/#/material/maps/default?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm)
+* [Product Page](https://www.syncfusion.com/angular-components/angular-maps?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm)
 
 ## Supported Frameworks
 
 Maps component is also offered in following list of frameworks.
 
-1. [Angular](https://www.npmjs.com/package/@syncfusion/ej2-ng-maps?utm_source=npm&utm_campaign=maps)
-2. [React](https://www.npmjs.com/package/@syncfusion/ej2-react-maps?utm_source=npm&utm_campaign=maps)
-3. [Vue.js](https://www.npmjs.com/package/@syncfusion/ej2-vue-maps?utm_source=npm&utm_campaign=maps)
-4. [ASP.NET Core](https://aspdotnetcore.syncfusion.com/Maps/Default#/material)
-5. [ASP.NET MVC](https://aspnetmvc.syncfusion.com/Maps/Default#/material)
-6. [JavaScript (ES5)](https://www.syncfusion.com/products/javascript/maps)
+1. [Angular](https://www.npmjs.com/package/@syncfusion/ej2-ng-maps?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm)
+2. [React](https://www.npmjs.com/package/@syncfusion/ej2-react-maps?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm)
+3. [Vue.js](https://www.npmjs.com/package/@syncfusion/ej2-vue-maps?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm)
+4. [ASP.NET Core](https://aspdotnetcore.syncfusion.com/Maps/Default#/material?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm)
+5. [ASP.NET MVC](https://aspnetmvc.syncfusion.com/Maps/Default#/material?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm)
```

Comment:
mvc & core links are broken. Please check that and remove

Reply:
removed